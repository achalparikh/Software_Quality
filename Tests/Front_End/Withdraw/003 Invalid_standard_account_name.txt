login
standard
CLARK KENT
withdraw
400.00
logout
