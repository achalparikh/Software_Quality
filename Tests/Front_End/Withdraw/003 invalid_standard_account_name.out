Enter user type - "standard" or "admin": 
Logging in as standard user - enter name: 
ERROR: User "CLARK KENT" does not exist. Try again 
ERROR: Not logged in
ERROR: Not logged in
You are now logged out
